### DISCUSSION TITLE
Uncaught Error: Class "Ssh" not found

### DISCUSSION THREAD

**Discussion by Webtica** [spatie/ssh#51] [Q&A] [answered]
Hi!

I just installed this package on the latest version 1.6.
When running the php file like this: 

`require 'vendor/autoload.php';
$process = Ssh::create('user', 'host')->execute('ls');
echo $process;`

I do get the error: Uncaught Error: Class "Ssh" not found. 
Any idea what this is? im using this in a plain PHP file/setup. 

Thanks

**Comment by paradoxos** [comment, marked as answer]
Had the same problem.
In my case there was a hidden space or some other character in the documentation.
I wrote it by hand instead of copy pasting it and worked.
